code/modules/projectiles/guns/projectile/automatic.dm

```diff
@@ -51,17 +51,17 @@
 
 /obj/item/weapon/gun/projectile/automatic/sts35
 	name = "assault rifle"
-	desc = "The rugged STS-35 is a durable automatic weapon of a make popular on the frontier worlds. Uses 5.56mm rounds. This one is unmarked."
+	desc = "The rugged STS-35 is a durable automatic weapon of a make popular on the frontier worlds. Uses 5.45mm rounds. This one is unmarked."

```

Comment:
Is it really necessary to say "this one is unmarked" also? I feel like this is just pointless fluff that doesn't add anything.

"This one is unmarked." Unmarked from what? What are other guns marked with? Serial numbers are not a feature in SS13.